src/App.js

```diff
@@ -1,14 +1,65 @@
 import React from "react";
-import "./App.css";
+import "./App.css"
 
 class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      counter: 0,
+      addScore: 1,
+      firstCount: 10,
+      firstAdd: 1,
+      secondAdd: 2,
+    };
```

Comment:
```suggestion
    this.state = {
      counter: 0,
      addScore: 1,
    };
```
You're storing a lot of things in state that you don't actually need. All you need is `counter` to keep track of the user's score, and `addScore` to keep track of how many points each click is worth. I'll refactor your custom methods too so that you can see what I mean.